docs: Update Generic scraper doc (#1941)
tomkerkhove/promitor: docs/configuration/v2.x/metrics/generic-azure-resource.md

You can declare to scrape a generic Azure resource via the `Generic` resource type.

> Promitor simplifies defining resource uris by using the subscription & resource

Reviewer: Nit
```suggestion
Promitor simplifies defining resource URIs by using the subscription & resource
```

  aggregation:
    type: Total
resources:
# Will scrape subscriptions/<sub>/resourceGroups/<rg>/providers/Microsoft.ServiceBus/namespaces/my-promitor-messaging
# Where <sub> & <rg> are coming from azureMetadata
- resourceUri: Microsoft.ServiceBus/namespaces/my-promitor-messaging
  filter: EntityName eq 'orders'

> # Will scrape subscriptions/sub2/resourceGroups/rg2/providers/Microsoft.ServiceBus/namespaces/my-other-promitor-messaging
> # Where sub2 & rg2 are coming from the definition of this resource.

Reviewer: Aligning with the above
```suggestion
# Will scrape subscriptions/<sub>/resourceGroups/<rg>/providers/Microsoft.ServiceBus/namespaces/my-other-promitor-messaging
# Where <sub> & <rg> are coming from the definition of this resource.
```

- resourceUri: Microsoft.ServiceBus/namespaces/my-other-promitor-messaging

>   resourceGroupName: rg2
>   subscriptionId: sub2

Reviewer: Providing a bit more info
```suggestion
  subscriptionId: example-subscription
  resourceGroupName: example-resource-group
```
